src/main/resources/test_files/runtime/InstanceOf.jq

```diff
@@ -25,7 +25,9 @@ null instance of null?,
 () instance of string+,
 () instance of string,
 () instance of (),
-hexBinary("aabbCC") instance of hexBinary
+hexBinary("aabbCC") instance of hexBinary,
+base64Binary("0 FB8 0F+9") instance of base64Binary,
```

Comment:
Something is not quite right, as these are invalid base64 encodings and these last two casts should throw errors (Zorba also doesn't accept this query). There can be no space in base64, only a-z, A-Z, 0-9, + / and =. A (here, large) base64 encoding/literal looks like this:

TWFuIGlzIGRpc3Rpbmd1aXNoZWQsIG5vdCBvbmx5IGJ5IGhpcyByZWFzb24sIGJ1dCBieSB0aGlz
IHNpbmd1bGFyIHBhc3Npb24gZnJvbSBvdGhlciBhbmltYWxzLCB3aGljaCBpcyBhIGx1c3Qgb2Yg
dGhlIG1pbmQsIHRoYXQgYnkgYSBwZXJzZXZlcmFuY2Ugb2YgZGVsaWdodCBpbiB0aGUgY29udGlu
dWVkIGFuZCBpbmRlZmF0aWdhYmxlIGdlbmVyYXRpb24gb2Yga25vd2xlZGdlLCBleGNlZWRzIHRo
ZSBzaG9ydCB2ZWhlbWVuY2Ugb2YgYW55IGNhcm5hbCBwbGVhc3VyZS4=

If there is an issue with DataTypeConverters, another option that is quite popular is org.apache.commons.codec.binary.Base64 with an encode and a decode function.

FYI a common practical mistake with base64 encoding libraries is that a string passed to and from a converter can be interpreted in two ways: as a base64 literal, or as an array of bytes (its characters). Many errors found in software amount to this confusion, as it can get easily tricky and the documentation is not always clear.